src/event_handler.cpp

```diff
@@ -297,6 +294,8 @@ void	EventHandler::Execute(const pollfd& entry, const std::string& msg) {
 }
 
 void EventHandler::ExecuteCommand(Event*& event_ptr) {
+	if (event_ptr->get_command() == message::kQuit)
+		Detach(event_ptr->get_fd());
 	database_.CheckEvent(event_ptr);
 	if (event_ptr->is_do_nothing())
 		return ;
```

Comment:
このタイミングでDetachは、問題はないのですが、もしQUITコマンドが失敗する可能性があった場合に、想定外の動作をする可能性があるので、最後に行う方が好ましいです。
(下の方が含められなかったので、提案をそのまま受け入れないでください。)
```suggestion
	database_.CheckEvent(event_ptr);
	if (event_ptr->is_do_nothing())
		return ;
	if (EventHandler::CheckNewChannel(*event_ptr))
		this->AddNewChannel(event_ptr);
	AddResponseMap(database_.ExecuteEvent(*event_ptr));
	database_.DeleteFinishedElements();
	if (!event_ptr->HasErrorOccurred() && event_ptr->get_command() == message::kQuit)
		Detach(event_ptr->get_fd());
```